variables.tf

```
@@ -603,3 +603,19 @@ variable "module_depends_on" {
   description = "(Optional) Define resources this module indirectly depends_on."
   default     = []
 }
+
+variable "workflow_disabled" {
```

I'd suggest a more general name, e.g.:
```suggestion
variable "github_actions_permissions" {
```